src/encoded/renderers.py

```diff
@@ -180,8 +187,13 @@ def security_tween(request):
                             # This header is parsed in renderer.js, or, more accurately,
                             # by libs/react-middleware.js which is imported by server.js and compiled into
                             # renderer.js. Is used to get access to User Info on initial web page render.
-                            response.headers['X-Request-JWT'] = request.cookies.get('jwtToken','')
-                            response.headers['X-User-Info'] = json.dumps(request.user_info) # Re-ified property set in authentication.py
+                            response.headers['X-Request-JWT'] = request.cookies.get('jwtToken', '')
+                            # TODO: Should user_info be copied before the del? If the user info is shared,
+                            #       we are modifying it for other uses. -kmp 24-Jan-2021
+                            user_info = request.user_info.copy()  # Re-ified property set in authentication.py
```

Comment:
@alexkb0009 and I decided a `.copy()` here was a good idea. I've made the corresponding change in `cgap-portal` too.